doctrine/annotations/1.10/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+    "conflict": {
+        "symfony/routing": "<6.0"
```

Comment:
Is this rule contradicting the title of this PR?

Reply:
this recipe is empty, but only when used with >=6.0, that's the point

Reply:
Right, thanks for your answer.